src/styles/_z-index.scss

```diff
@@ -19,6 +19,6 @@ $z-index-slideup-root: 150;
 
 $z-index-popover-root: 200;
 
-$z-index-dev-portal-top-links: 3;
-$z-index-dev-portal-breadcrumb: 2;
```

Comment:
this was the reason 👇 for keeping high z-index

![image](https://user-images.githubusercontent.com/29539278/132897823-8fa2250a-419b-4d2d-854a-11aa9834355c.png)

Reply:
To reproduce this:
1. open any project details page.
2. test on a device with a small height, or simply open dev tools at the bottom.

Reply:
ah ok thanks for spotting it, i have increased the z-index range of values to the hundreds to allow more room to work with. Fixed this as well